source/administration/devops-command-center.rst

```diff
@@ -9,159 +9,95 @@ DevOps Command Center
 
-Viewing and creating playbooks
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Starting an incident
+~~~~~~~~~~~~~~~~~~~~
 
-Select **!** in the channel header to open the RHS panel. Select the **Playbooks** icon to open the playbooks backstage and list all playbooks associated with the current team. Select **+ New Playbook** to start a new playbook and checklist. Enter a playbook name and choose **Save Playbook**.
+To start an incident, use one of the following steps:
 
-Creating checklists
-~~~~~~~~~~~~~~~~~~~
+- Use the slash command */incident start* from any channel.
+- Select the shield icon in the channel header, and click **+ Start Incident**
+- Use the context menu of a post and select **Start incident**.
 
-Checklists are created within a playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a playbook, or during an incident.
+An interactive dialog appears prompting the selection of a playbook and channel name. Click **Start Incident** to create an incident with the selected playbook. Only playbook members can use the playbook to start an incident.
 
-Editing checklists
-~~~~~~~~~~~~~~~~~~~
+The newly-created incident channel is the central place for discussion related to the incident. The incident bot announces the creator of the incident with a post in the channel. If an incident is started from the context menu of a post, the text of that post is included in the announcement message.
 
-You can add steps and edit checklist items during an active incident. You can also edit checklist items in playbooks that are not being used in active incidents.
+The creator of a playbook is automatically added as the first member and becomes the commander. The commander is responsible for adding other members to the incident as needed, and may assign a new commander once a member is added to the channel. To change commanders, click the current commander's name in the RHS and select the new commander. Only members of the channel may be selected as commanders. To change commander to a user who is not in the channel, first add the user to the channel. 
 
-To edit a checklist, choose **Edit**.
+Interacting with an incident
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Select the text field to edit the item details.
-- Hover over an input box and select the **X** to remove the checklist item.
-- Drag the hamburger menu to rearrange the checklist items.
+Browsing to an incident channel automatically opens the RHS showing the current commander, duration, active stage and corresponding steps. Any member of the user is free to change the commander, view other stages, mark a new stage as active, or mark a step as completed.
 
-Choose **Done** to save the changes.
+Some steps may be assigned slash commands which can be run at any time by clicking **(Run)** next to the slash command. Use slash commands to expedite the completion of a step, such as inviting a user to the channel or triggering some third-party integration.
```

Comment:
Can we add a description of the new feature for assigning items to members, or is that out of scope? Something like
"Each step can be optionally assigned to a member of the incident. Click on the **No assignee** label below the step title to select the assignee."